common/src/main/scala/uk/ac/wellcome/models/Work.scala

```diff
@@ -12,9 +13,13 @@ case class SourceIdentifier(source: String, sourceId: String, value: String)
 case class IdentifiedWork(canonicalId: String, work: Work)
 
 /** A representation of an item in our ontology, without a canonical identifier */
-case class Work(identifiers: List[SourceIdentifier],
-                       label: String,
-                       accessStatus: Option[String] = None)
+case class Work(
+  identifiers: List[SourceIdentifier],
+  label: String,
+  accessStatus: Option[String] = None
+) {
+  @JsonProperty("type") val ld_type: String = "Work"
```

Comment:
Scala convention is camelCase right? We should follow the convention.

_even if underscores are more readable_